docs/docs/mdx/markdown-syntax.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Markdown Syntax
+---
+
+### Headings
+
+```
+# heading 1
+## heading 2
+### heading 3
+#### heading 4
+##### heading 5
+###### heading 6
+```
+
+# heading 1
+
+## heading 2
+
+### heading 3
+
+#### heading 4
+
+##### heading 5
+
+###### heading 6
+
+- each heading gets converted to their HTML equivalent
+  - i.e. `# heading 1` is `<h1>heading 1</h1>`
+- Correct usage of each heading should follow the
+  [accessibility guidelines](https://www.w3.org/WAI/tutorials/page-structure/headings/) set by the World Wide Web Consortium (W3C)
+
+### Emphasized text
+
+- Italic
+  - one asterisk or one underscore
+    - `*italic*` or `_italic_`
+    - _italic!_
+- Bold
+  - two asterisks or two underscores
+    - `**bold**` or `__bold__`
+    - **bold!**
+- Italic and Bold
+
+  - three asterisks or three underscore
+    - `***italic and bold***` or `___italic and bold___`
+    - **_italic and bold!!_**
+
+### Lists
+
+- Unordered
```

Comment:
```suggestion
### Unordered
```
Recommending a heading here since the list format is a little _too_ meta when talking about lists! 🤯